client/app/app.js

```diff
@@ -16,10 +16,12 @@ App = Ember.Application.extend({
 loadInitializers(App, config.modulePrefix);
 
 // make bootstrap close the menu after click on an item.
+// jshint ignore: start
 $(document).on('click','.navbar-collapse.in',function(e) {
```

Comment:
This needs to move to the main-nav component, so ignoring the jshint for now, but will move this soon.